playwright/test/search-stories.test.ts

```diff
@@ -0,0 +1,87 @@
+import { test, expect } from '@playwright/test';
+import { newSearch } from './helpers/contexts';
+import {
+  locateAndConfirmContributorInfoMatchesStory,
+  navigateToNextPageAndConfirmNavigation,
+  navigateToStoryResultAndConfirmTitleMatches,
+  searchQuerySubmitAndWait,
+  selectAndWaitForFilter,
+  testIfFilterIsApplied,
+} from './helpers/search';
+
+test.describe.configure({ mode: 'parallel' });
+
+test('(1) | The user can search for instances of a topic and format their results by type and contributor', async ({
+  page,
+  context,
+}) => {
+  await newSearch(context, page, 'stories');
+  await searchQuerySubmitAndWait('milk', page);
+  await selectAndWaitForFilter('Formats', 'W7TfJRAAAJ1D0eLK', page); // Articles
+  await testIfFilterIsApplied('Article', page);
+  await selectAndWaitForFilter('Contributors', 'WfLM2yoAAKKpVrxD', page);
+  // Contributor (AW)
+  await testIfFilterIsApplied('Alice White', page);
+  await navigateToStoryResultAndConfirmTitleMatches(1, page);
+});
+
+test(`(2) | The user can see the correct contributor's name below the story title in search results`, async ({
+  page,
+  context,
+}) => {
+  await newSearch(context, page, 'stories');
+  await searchQuerySubmitAndWait('medieval doodles', page);
+  // In case of similarly titled article
+  await selectAndWaitForFilter('Contributors', 'XIp1ExAAAPyQB4NN', page);
```

Comment:
If you want to search for a specific story, you can search them by ID (so query `XLRmEBEAABp4vDEG`)! Just realising this as I'm typing that it's actually quite useful for tests like this.

Reply:
That was my thinking - just popping the article tidies this up nicely